tests/handler_tests/test_mysql_handler.py

```diff
@@ -136,8 +136,20 @@ def test_create_table(self, handler):
         tables = self.get_table_names(handler)
         assert new_table in tables, f"expected to have {new_table} in database, but got: {tables}"
 
+    def test_insert_table(self, handler):
+        tablename = "test_mdb"
+        res = handler.native_query(f"INSERT INTO {new_table} (test_col) values (1), (2), (3))")
+        self.check_valid_response(res)
+        handler.disconnect()
+        handler.connect()
+        res = handler.query(f"SELECT count(*) FROM {new_table}")
+        self.check_valid_response(res)
+        got_rows = res.data_frame.shape[0]
+        assert got_rows == 3
+
+
     def test_drop_table(self, handler):
-        drop_table = "test_md"
```

Comment:
I am guessing that this is a typo?

Reply:
It looks so, anyway we will need to change this tests